lxd/resource_lxd_container.go

```diff
@@ -222,6 +229,11 @@ func resourceLxdContainerCreate(d *schema.ResourceData, meta interface{}) error
 	}
 	refreshInterval := meta.(*lxdProvider).RefreshInterval
 
+	project := d.Get("project")
+	if project != nil {
```

Comment:
I think we should handle this similarly to the provider: only pass a project if the value is not `""`.